spacelift/resource_audit_trail_webhook.go

```diff
@@ -47,7 +47,12 @@ func resourceAuditTrailWebhook() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 				Sensitive:   true,
-				Description: "`secret` is a secret that Spacelift will send with the request",
+				Description: "`secret` is a secret that Spacelift will send with the request.",
+			},
+			"custom_headers": {
+				Type:        schema.TypeMap,
+				Optional:    true,
+				Description: "`custom_headers` is a Map of key values strings, that will be passed as headers with audit trail call.",
```

Comment:
I wonder if this should be "with the audit trail call", or "with audit trail calls"? Maybe we should use the word "request" instead of "call" as well?